pkg/internal/testhelpers/containers.go

```diff
@@ -130,24 +236,145 @@ func (s stdoutLogConsumer) Accept(l testcontainers.Log) {
 }
 
 // StartPGContainer starts a postgreSQL container for use in testing
-func StartPGContainer(ctx context.Context, usePromscaleExtension bool, useTimescaleDB bool, useTimescale2 bool, testDataDir string, printLogs bool) (testcontainers.Container, error) {
+func StartPGContainer(
+	ctx context.Context,
+	extensionState ExtensionState,
+	testDataDir string,
+	printLogs bool,
+) (testcontainers.Container, io.Closer, error) {
 	var image string
-	if usePromscaleExtension && useTimescale2 {
+	switch extensionState {
+	case MultinodeAndPromscale:
+		image = "timescaledev/promscale-extension:2.0.0-rc2-pg12"
+	case Multinode:
+		image = "timescale/timescaledb:2.0.0-rc2-pg12"
+	case Timescale2AndPromscale:
 		image = "timescaledev/promscale-extension:2.0.0-rc2-pg12"
-	} else if useTimescale2 {
+	case Timescale2:
 		image = "timescale/timescaledb:2.0.0-rc2-pg12"
-	} else if usePromscaleExtension && useTimescaleDB {
+	case Timescale1AndPromscale:
 		image = "timescaledev/promscale-extension:latest-pg12"
-	} else if useTimescaleDB {
+	case Timescale1:
 		image = "timescale/timescaledb:latest-pg12"
-	} else {
+	case VanillaPostgres:
 		image = "postgres:12"
 	}
-	return StartPGContainerWithImage(ctx, image, testDataDir, "", printLogs, usePromscaleExtension, useTimescaleDB)
+
+	return StartPGContainerWithImage(ctx, image, testDataDir, "", printLogs, extensionState)
 }
 
-func StartPGContainerWithImage(ctx context.Context, image string, testDataDir string, dataDir string, printLogs bool, usePromscaleExtension bool, useTimescaleDB bool) (testcontainers.Container, error) {
+func StartPGContainerWithImage(ctx context.Context,
```

Comment:
This now starts multiple containers. The name should reflect that
```suggestion
func StartPGClusterWithImage(ctx context.Context,
```

Reply:
hmm, it's not always a cluster though. How about `StartDatabaseWithImage(...)`